### PR TITLE
Added basic logging and response timing

### PR DESCRIPTION
**Pull request by graytonio**
Also created option in application to turn off logging

### CODE REVIEW

File: application.ts [hunk 1]
```diff
@@ -3,21 +3,26 @@ import { Context } from "./context.ts";
 import { Router } from "./router/router.ts";
 
 type ServerConfig = {
-  port: number;
+  port?: number;
+  verbose?: boolean;
```
Comment (disizali) [inline application.ts]: very nice located configuration.

File: application.ts [hunk 2]
```diff
@@ -3,21 +3,26 @@ import { Context } from "./context.ts";
...
 };
 
 
 export class Application {
+  private verbose: boolean = true;
```
Comment (disizali) [inline application.ts]:
```suggestion
  private verbose: boolean;
```

File: application.ts [hunk 3]
```diff
@@ -40,7 +45,11 @@ export class Application {
         }
       );
       if (route) {
+        let t0 = performance.now();
         route.handler(new Context(request));
+        let t1 = performance.now();
+
+        if(this.verbose) console.log(`${request.method}: ${request.conn.remoteAddr.hostname}${request.url} - ${(t1 - t0).toFixed(2)}ms`);
```
Comment (disizali) [inline application.ts]: very useful.

**Review by disizali — CHANGES_REQUESTED** [review]
glad to see your cooperation, thanks @graytonio!
can you just rename `verbose` to something like `log` for now, please?
we have plans to have variant log levels, so it make more sense.

**Comment by graytonio** [comment]
Renamed Verbose to Log @disizali

**Comment by disizali** [comment]
LGTM, thanks again!

**Pull request merged**